ert_gui/simulation/simulation_panel.py

```diff
@@ -119,9 +119,9 @@ def getSimulationArguments(self):
     def runSimulation(self):
         case_name = self.facade.get_current_case_name()
         message = (
-            "Are you sure you want to use case '%s' for initialization of the initial ensemble when running the simulations?"
-            % case_name
-        )
+            "Are you sure you want to use case '%s' for initialization of "
+            "the initial ensemble when running the simulations?"
+        ) % case_name
```

Comment:
Convert to f string since we are touching it anyway?

Reply:
```
$ pylint res ert_shared ert_gui ert_data/ | grep "could be a f-string" | wc -l
437

```

🤔  maybe that warrants its own PR..